pkgs/applications/audio/bespokesynth/default.nix

```diff
@@ -133,23 +131,27 @@ stdenv.mkDerivation rec {
         --prefix PATH : '${lib.makeBinPath [
           gnome.zenity
           (python3.withPackages (ps: with ps; [ jedi ]))
-        ]}' \
-        --prefix LD_LIBRARY_PATH : '${lib.makeLibraryPath [
-          libXrandr
-          libXinerama
-          libXcursor
-          libXScrnSaver
         ]}'
     '';
 
+  env.NIX_LDFLAGS = "-rpath ${lib.makeLibraryPath ([
```

Comment:
```suggestion
  env.NIX_LDFLAGS = lib.optionalString stdenv.hostPlatform.isLinux "-rpath ${lib.makeLibraryPath ([
```

Otherwise this pulls in all of the X11 stuff on Darwin.
![image](https://github.com/NixOS/nixpkgs/assets/23431373/bba8cdbe-4eff-4f16-9bdd-bb48ae4450ab)